docs/src/development/troubleshoot.md

```diff
@@ -26,8 +26,6 @@ highlight=false
 1. In the management console, navigate to the environment you want to redeploy.
 1. Click **Redeploy**.
 
-![Redeploy button in Console](/images/troubleshoot/platformsh-redeploy-button.png "0.5")
```

Comment:
I've removed this image because it looks awkward inside a codetab and it's unnecessary.

Reply:
It looks awkward because it seems the resizing thing doesn't work within the code tab. That's something that should be fixed, so I opened #2016. But if you think it's unnecessary here, it's fine to remove it anyway. Just a bug to try to fix for the future.